CODE_OF_CONDUCT.md

```diff
@@ -1,5 +1,149 @@
 # Code of Conduct
 
-The Node.js Code of Conduct document has moved to
-https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md. Please update
-links to this document accordingly.
+## When Something Happens
+
+If you see a Code of Conduct violation, follow these steps:
+
+1. Let the person know that what they did is not appropriate and ask them to stop and/or edit their message(s) or commits.
```

Comment:
is it worth adding something along the lines of "let them know why their actions/behaviour were inappropriate" ?

Reply:
@sandfox I'm wary of making seem like it's the person on the receiving end who's responsible for explaining and justifying that in the moment. In my experience some sort of explanation tends to happen, but it might come a while after the incident and from someone else

Reply:
@SomeHats completely agree that they shouldn't have to be responsible for justifying their hurt as in some cases it's just exhausting or causes further pain (by intent). I think I just want to try to nudge people towards communicating if they can/wish, but if you think this doesn't need spelling out so explicitly, or is going to used against it's spirit then quite happy to leave Kat's text as is.